web/src/components/vis/VisPca.vue

```diff
@@ -164,20 +186,20 @@ export default {
 
       const eigval = [
         trace / 2 + Math.sqrt(trace * trace / 4 - det),
-        trace / 2 - Math.sqrt(trace * trace / 4 - det)
+        trace / 2 - Math.sqrt(trace * trace / 4 - det),
       ];
 
-      const eigvec = Math.abs(xy) < 1e-10 ? [[1, 0], [0, 1]] :
-        [[eigval[0] - yy, xy],
-         [eigval[1] - yy, xy]];
+      const eigvec = Math.abs(xy) < 1e-10 ? [[1, 0], [0, 1]]
```

Comment:
The colon on the next line is better; should we move the identity matrix expression to its own line too? Or, in general, are we better off dumping the `?:` operator here?

Reply:
I'm a really bad offender of overusing ternary, so I'm probably not the one to ask.  I do like the look of `? / :` each on their own line personally, but have never checked official style

Reply:
Let's not overthink it and just leave this as it is now :cat: